managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerConfig.java

```diff
@@ -56,6 +56,7 @@
     private boolean autoSkipNonRecoverableData;
     private long offloadLedgerDeletionLagMs = TimeUnit.HOURS.toMillis(4);
     private long offloadAutoTriggerSizeThresholdBytes = -1;
+    private long opTimeoutSec = 5 * 60; //5 minutes
```

Comment:
Maybe use a more explicit name like `metadataOperationsTimeoutSeconds` ? 

Also, the default in `ServiceConfiguration` seems to be 60sec

Reply:
sure, let me rename it. 

> Also, the default in ServiceConfiguration seems to be 60sec
Sure, I can make it 60 sec but in this PR, default time we are setting as `ZooKeeperSessionTimeout (default 30sec) *2` as it mainly happens when zk-client fails to complete callback when zk leader restarts.